exercises/rotational-cipher/rotational_cipher.exs

```diff
@@ -0,0 +1,13 @@
+defmodule RotationalCipher do
+  @doc """
+  Given a plaintext and amount to shift by, return a rotated string.
+
+  Example:
+  iex> RotationalCipher.rotate("Attack at dawn", 13)
+  "Nggnpx ng qnja"
+  """
+  @spec rotate(text :: String.t(), shift :: integer) :: String.t()
+  def rotate(_text, _shift) do
```

Comment:
Guess it was just warnings in iex after all. Removed them from this and poker, and the warning in iex at least tells users there's something they need to fix.